src/ResourceManager/ManagedServiceIdentity/ManagedServiceIdentity.sln

```diff
@@ -4,6 +4,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 VisualStudioVersion = 14.0.25420.1
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Commands.ManagedServiceIdentity", "Commands.ManagedServiceIdentity\Commands.ManagedServiceIdentity.csproj", "{228EB071-FA04-43B3-95F9-7D76DBF0B850}"
+	ProjectSection(ProjectDependencies) = postProject
```

Comment:
This doesn't look correct - please fix this so it follows the format below.  Did you add the new project on visual studio?